lib/program.js

```diff
@@ -1,16 +1,79 @@
 #!/usr/bin/env node
 import program from 'commander';
+import chalk from 'chalk';
+import Config from 'bcfg';
+const { execSync } = require('child_process');
+const Logger = require('blgr');
+
+import create from './create';
+import install from './install';
+import uninstall from './uninstall';
 
-import makePlugin from './makePlugin';
 const { description, version } = require('../package.json');
 
-program
-  .version(version)
-  .description(description);
+const logger = new Logger('info');
+
+const config = new Config('bpanel');
+config.load({
+  env: true,
+  argv: true,
+  arg: true
+});
+
+(async function() {
+  await logger.open();
+
+  program
```

Comment:
does `bcfg` have cli tools built in? I remember seeing something with how nodar used it for the handshake address generator

Reply:
This would allow for us to eliminate a dependency. It may be good to learn how to use our tool to build cli tools, but also there could be more important things to do

Reply:
It doesn't really have CLI tools per se, just the ability to parse options from the command line. It would be possible to do this with bcfg but it would require building out some of the scaffolding that `commander` and `inquirer` provide out of the box, e.g. options parsing, help menu, and the convenience methods that allow the code to be a little more declarative. I'm not against using bcfg, but for what it's worth, bcfg was in the process of being separated out when this was first built, so migrating over to it would require a decent amount of refactoring.

Reply:
Ok, makes sense, this looks good to me